ui/src/main/webapp/src/app/executions/executions-list.component.html

```diff
@@ -30,7 +30,16 @@
         </thead>
         <tbody>
         <tr *ngFor="let execution of sortedExecutions" [class]="getClass(execution)" class="execution-row">
-            <td><a [routerLink]="['/projects', execution.projectId, execution.id, 'execution-details']">#{{execution.id}}</a></td>
+            <td>
+                <span *ngIf="execution.state == 'COMPLETED' && execution?.analysisContext?.generateStaticReports; else elseExecutionState">
+                    <a  class="pointer link" target="_blank" href="{{formatStaticReportUrl(execution)}}" title="Show reports">
+                        #{{execution.id}} <i class="fa fa-external-link fa-fw"></i>
```

Comment:
Here it is the screenshot with 3 examples of "icon before" and 4 of "icon after"
![screenshot from 2017-06-27 10-14-04](https://user-images.githubusercontent.com/7288588/27577590-9868a4fe-5b21-11e7-8087-4369420cc4a6.png)